src/shogun-core-main/src/main/java/de/terrestris/shoguncore/service/EndpointDocService.java

```diff
@@ -26,12 +25,7 @@
      */
     @PreAuthorize("hasRole(@configHolder.getSuperAdminRoleName())")
     public Set<RequestMappingInfo> getEndpoints(RequestMappingHandlerMapping requestMappingHandlerMapping) {
-
-        if (requestMappingHandlerMapping.getHandlerMethods() != null) {
-            return requestMappingHandlerMapping.getHandlerMethods().keySet();
-        }
-
-        return null;
+        return requestMappingHandlerMapping.getHandlerMethods().keySet();
```

Comment:
Doesn't this now blow up in case `requestMappingHandlerMapping.getHandlerMethods()` is `null`?

Reply:
No, it doesn't since `getHandlerMethods` never returns null in any case.